modules/deploy/pages/deployment-option/self-hosted/kubernetes/cloud-instance-local-storage.adoc

```diff
@@ -0,0 +1,118 @@
+
+NOTE: For all recommended instance types, choose a model that meets the xref:./kubernetes-cluster-requirements.adoc[requirements and recommendations], depending on the size of your Redpanda cluster and your expected workload.
+
+== Amazon EKS
+
+*Default filesystem*: EKS defaults to the ext4 filesystem. Use XFS instead where possible.
+
+*Recommended instance types*:
+
+- General purpose: General-purpose instances provide a balance of compute, memory and networking resources, and can be used for a variety of diverse workloads.
+
+** https://aws.amazon.com/ec2/instance-types/m5/[M5d^]
+** https://aws.amazon.com/ec2/instance-types/m5/[M5ad^]
+** https://aws.amazon.com/ec2/instance-types/m5/[M5dn^]
+** https://aws.amazon.com/ec2/instance-types/m6g/[M6gd^]
+** https://aws.amazon.com/ec2/instance-types/m7g/[M7gd^]
+
+- Memory optimized: Memory-optimized instances are designed to deliver fast performance for workloads that process large data sets in memory.
+
+** https://aws.amazon.com/ec2/instance-types/r5/[R5ad^]
+** https://aws.amazon.com/ec2/instance-types/r5/[R5d^]
+** https://aws.amazon.com/ec2/instance-types/r5/[R5dn^]
+** https://aws.amazon.com/ec2/instance-types/r6g/[R6gd^]
+** https://aws.amazon.com/ec2/instance-types/r6i/[R6id^]
+** https://aws.amazon.com/ec2/instance-types/r6i/[R6idn^]
+** https://aws.amazon.com/ec2/instance-types/r7g/[R7gd^]
+** https://aws.amazon.com/ec2/instance-types/x2/[X2gd^]
+** https://aws.amazon.com/ec2/instance-types/x2i/[X2idn^]
+** https://aws.amazon.com/ec2/instance-types/x2i/[X2iedn^]
+** https://aws.amazon.com/ec2/instance-types/z1d/[z1d^]
+
+- Storage optimized: Storage-optimized instances are designed for workloads that require high, sequential read and write access to very large data sets on local storage. They are optimized to deliver tens of thousands of low-latency, random I/O operations per second (IOPS) to applications.
+
+** https://aws.amazon.com/ec2/instance-types/i3/[I3^]
+** https://aws.amazon.com/ec2/instance-types/i3en/[I3en^]
+
+- Compute optimized: Compute-optimized instances deliver cost-effective high performance at a low price per compute ratio for running advanced compute-intensive workloads.
+
+** https://aws.amazon.com/ec2/instance-types/c5/[C5d^]
+** https://aws.amazon.com/ec2/instance-types/c5/[C5ad^]
+
+== Azure AKS
+
+*Filesystem*: AKS often defaults to the ext4 filesystem. Use XFS instead where possible.
+
+*Recommended machine types*:
+
+- Memory optimized: Memory-optimized VM sizes offer a high memory-to-CPU ratio.
+
+** https://learn.microsoft.com/en-us/azure/virtual-machines/ebdsv5-ebsv5-series[Ebsv5 series (NVMe)^]
+** https://learn.microsoft.com/en-us/azure/virtual-machines/ebdsv5-ebsv5-series[Ebdsv5 series (NVMe)^]
+
+- Storage optimized: Storage-optimized virtual machine (VM) sizes offer high disk throughput and IO.
```

Comment:
There are 2 places in this file that define virtual machine (VM). Suggest only putting this in the first instance of it above.